src/index.ts

```diff
@@ -3,25 +3,38 @@ import readline from 'readline'
 import os from 'os'
 import fs from 'fs'
 
+const platform = process.platform
+
 const askQuestion = (query: string) => {
   const rl = readline.createInterface({
     input: process.stdin,
     output: process.stdout
   })
 
-  const readAnswer = (ans: string) => {
+  const readAnswerDarwin = (ans: string) => {
     const data = `@platformbuilders:registry=https://npm.pkg.github.com/\n//npm.pkg.github.com/:_authToken=${ans}\n`
     fs.appendFileSync(`/Users/${os.userInfo().username}/.npmrc`, data)
```

Comment:
Sugestão, ao invés de duplicar a função poderia passar outro argumento ex: `path` com valores diferentes por plataforma como para o linux `/home` e osx `/Users`

Reply:
Ok, se eu for prosseguir com essa abordagem, o comentário de cima deve ser ignorado, certo?!

Se for pra optar entre as duas abordagens eu seguiria com essa, o que você acha?